elasticsearch/utils/es_util_env

```diff
@@ -6,3 +6,18 @@ fi
 
 ES_BASE=${ES_BASE:-https://localhost:9200}
 curl_get='curl -s -X GET --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key'
+
+
+get_security_conf_index() {
+  local es_config_file=${ES_PATH_CONF}/elasticsearch.yml
+  local index=$( python -c 'import sys,yaml
+hsh = yaml.load(open(sys.argv[1]))
+if "searchguard" in hsh and "config_index_name" in hsh["searchguard"]:
```

Comment:
i suspect we don't need this since without changes we dynamically init the security index and we can use the OD default which i believe is actually 'security'

Reply:
We need this for now, until everything is using es6 and es6 components